add-on/chrome/modules/MozLoopAPI.jsm

```diff
@@ -1001,7 +1002,48 @@ const kMessageHandlers = {
    */
   TelemetryAddValue: function(message, reply) {
     let [histogramId, value] = message.data;
-    Services.telemetry.getHistogramById(histogramId).add(value);
+    if (histogramId === "LOOP_MAU") {
+      let buckets;
+      let pref = "mau.";
+      this.GetAllConstants(null result => buckets = result.LOOP_MAU_TYPE);
+      switch(value) {
+        case buckets.OPEN_PANEL:
+          pref += "openPanel";
+          break;
+        case buckets.OPEN_CONVERSATION:
+          pref += "openConversation";
+          break;
+        case buckets.ROOM_OPEN:
+          pref += "roomOpen";
+          break;
+        case buckets.ROOM_SHARE:
+          pref += "roomShare";
+          break;
+        case buckets.ROOM_DELETE:
+          pref += "roomDelete";
+          break;
+      }
+
+      LoopAPI.sendMessageToHandler({
+        name: "GetLoopPref",
+        data: [pref]
+      }, result => {
+        var delta = new Date() - new Date(pref);
+        // Send telemetry event if period (30 days) passed.
+        // 0 is default value for pref.
+        // 2592000 seconds in 30 days
+        if (pref === 0 || delta >= 2592000 * 1000) {
+          Services.telemetry.getHistogramById(histogramId).add(value);
+          let timestamp = Math.floor(new Date().getTime() / 1000);
+          LoopAPI.sendMessageToHandler({
+            name: "SetLoopPref",
+            data: [pref, timestamp]
+          });
+        }
+      });
```

Comment:
I'd also really appreciate if you'd put the `browser_mozLoop_telemetry.js` changes in this PR... even though you can't test them here :wink: